packages/ra-dependent-input/README.md

```diff
@@ -0,0 +1,215 @@
+# ra-dependent-input
+
+A component for displaying inputs and fields depending on other inputs or fields values in [react-admin](https://github.com/marmelab/react-admin).
```

Comment:
Introduces two new components for react-admin: `<DependentField>` and `<DependentInput>`. Use these components di display inputs and fields depending on other inputs or fields values